### PR TITLE
Class22 131 feature add partial functionality on shopping cart page

### PR DESCRIPTION
**Pull request by viedamayanti**
# Description:

Add functionality for checkbox, dynamic cart (sum total) and remove btn

# Jira:

https://hackyourfuture-dk.atlassian.net/jira/software/projects/CLASS22/boards/5?selectedIssue=CLASS22-131

### CODE REVIEW

File: packages/client/src/containers/ShoppingCartPage/ShoppingCartPage.container.js [hunk 1]
```diff
@@ -10,34 +10,45 @@ import { useNavigate } from 'react-router-dom';
...
   const navigate = useNavigate();
-
-  const products = [
+  const [products, setProducts] = useState([
     {
       id: 1,
       description: 'Low waist Jeans',
       amount: 199.99,
       quantity: 1,
+      selected: false,
```
Comment (shpomp) [inline packages/client/src/containers/ShoppingCartPage/ShoppingCartPage.container.js]: This is unfortunately unsustainable 🙏🏽 
We do not and will not have a property of "selected" in any kind of API response once we fetch the shopping cart content from the DB, so refactoring the the fetch would  break your whole "selected" logic.

**Review by shpomp — CHANGES_REQUESTED** [review]

File: packages/client/src/containers/ShoppingCartPage/ShoppingCartPage.container.js (hunk 1 above)
Comment (viedamayanti) [reply packages/client/src/containers/ShoppingCartPage/ShoppingCartPage.container.js]: Hi Karo yes I already made changes for that

**Review by shpomp — APPROVED** [review]

File: packages/client/src/containers/ShoppingCartPage/ShoppingCartPage.container.js (hunk 1 above)
Comment (miltonhultgren) [reply packages/client/src/containers/ShoppingCartPage/ShoppingCartPage.container.js]: Why are we even storing/fetching the shopping cart state from the database?

Orders that are still being made by the user should not live in your database, they should live in local storage or a similar client side temporary storage.

Imagine if 100 customers start making orders by adding items but never finish their orders? Over time you'll get more and more half ready orders in your database to clean up.

Comment (shpomp) [reply packages/client/src/containers/ShoppingCartPage/ShoppingCartPage.container.js]: @miltonhultgren we will go with DB for this project. At some point there was a bit of a clash and discussion about local/db and how to make a unified process without blocking each other, and we landed on the shopping cart living in the db with a specific order status. 
Workarounds 😓

**Pull request merged**